main/build.gradle

```diff
@@ -408,6 +408,10 @@ dependencies {
 
     // Zxing barcode reader integration
     implementation 'com.google.zxing:android-integration:3.3.0'
+
+    // MarkDown view
+    def markwonVersion = '4.6.2';
```

Comment:
Typo: markwon -> markdown

Reply:
Sorry. Just realized that the library is called "markwon"

Reply:
actually no, in this case, as the library calls itself "markwon"